examples/features/vl3-lb/README.md

```diff
@@ -0,0 +1,79 @@
+# vL3 Load Balancer
+
+This example shows what is a vL3 Load Balancer (LB) and how it works.
+
+## Description
+
+We all know and use Kubernetes Services in our work. 
+Service is a method for exposing a network application that is running as one or more Pods in your cluster. It distributes traffic across a set of selected Pods.
+
+To define a Kubernetes Service:
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: my-service
+spec:
+  selector:
+    app.kubernetes.io/name: MyApp
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 9376
+```
+Now you can reach the service by its name.
+
+It turned out that we can use a similar mechanism in our vL3 networks - we can use a **vL3 Load Balancer**.
+Being on the same vL3 network, clients can behave like pods - they can communicate with each other. Using selectors, we can combine some of them to implement a service.
+vL3-LB will help us with this. We will call these clients that implement the service as _"real servers"_.
+
+So, vL3-LB has the following parameters:
+
+NSM env | Description
+- | -
+`NSM_SELECTOR`    | labels that group the vL3 clients we need to implement any service
+`NSM_PROTOCOL`    | TCP or UDP IP protocol
+`NSM_PORT`        | LB port
+`NSM_TARGET_PORT` | real server port
```

Comment:
In markdown view of the file this looks as a single line.
I guess you want to convert this into a table.

Reply:
Yes.
Thanks, fixed